src/Tizen.NUI.Components/Controls/SelectGroup.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/*
- * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ * Copyright(c) 2019-2021 Samsung Electronics Co., Ltd.
```

Comment:
Why don't we not use 2021 only?

Reply:
I was guided to update copyright with a range rather than a single year in another tizen project.
I have searched for it. Some articles indicate that removing the original year can cause problems, so I think we need to discuss it with other members.

> The copyright notice on a work establishes a claim to copyright. The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date and that means if somebody has copied the work in the meantime and they claim its theirs on the ground that their publishing the copy was before your claim, then it will be difficult to establish who is the originator of the work.

Reply:
Oh... thank you for your good information.